es60x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala

```diff
@@ -125,11 +130,15 @@ class ReadonlyRestPlugin(s: Settings, p: Path)
                                  xContentRegistry: NamedXContentRegistry,
                                  networkService: NetworkService,
                                  dispatcher: HttpServerTransport.Dispatcher): util.Map[String, Supplier[HttpServerTransport]] = {
+    val fipsCompliant = fipsConfig.fipsMode match {
```

Comment:
maybe we can wrap the code into IsFibsCompliant typeclass and place next to FipsConfiguration class?